flight/odlc_boundaries/search_path.py

```diff
@@ -0,0 +1,205 @@
+        if health.is_global_position_ok:
+            logging.info("Global position estimate ok")
+            break
+
+    logging.info("-- Arming")
+    await drone.action.arm()
+
+    logging.info("-- Taking off")
+    await drone.action.takeoff()
+
+    # wait for drone to take off
+    await asyncio.sleep(10)
+
+    # move to each waypoint in mission
+    for point in range(3):
+        await move_to(drone, Waypoint["lats"][point], Waypoint["longs"][point], 85, True)
+
+    # infinite loop till forced disconnect
+    while True:
+        await asyncio.sleep(1)
+
+    # Official Coordinates for Maryland
+
+
+if __name__ == "__main__":
+    data_search_area_boundary: list[dict[str, float]] = [
+        {"latitude": 38.3144070396263, "longitude": -76.54394394383165},  # Top Right Corner
+        {"latitude": 38.31430872867596, "longitude": -76.54397320409971},  # Right Midpoint
+        {"latitude": 38.31421041772561, "longitude": -76.54400246436776},  # Bottom Right Corner
+        {"latitude": 38.31461622313521, "longitude": -76.54516993186949},  # Top Left Corner
+        {"latitude": 38.31451966813249, "longitude": -76.54519982319357},  # Left Midpoint
+        {"latitude": 38.31442311312976, "longitude": -76.54522971451763},  # Bottom Left Corner
+    ]
+
+    # Test Coordinates at the Golf Course
+
+    # data_search_area_boundary: List[Dict[str, float]] = [
+    #   {"latitude": 37.94949210234766, "longitude": -91.78461752885457,}, # Top Left Corner
+    #   {"latitude": 37.94890371012351, "longitude": -91.78484629708285,}, # Bottom Left Corner
+    #   {"latitude": 37.94824920904741, "longitude": -91.78307727392504,}, # Bottom Right Corner
+    #   {"latitude": 37.94900523272037, "longitude": -91.782976748332,}, # Top Right Corner
+    #   {"latitude": 37.94949210234766, "longitude": -91.78461752885457,}, # Top Left Corner
+    #   {"latitude": 37.94919790623559, "longitude": -91.78473191296871}, # Left Midpoint
+    #   {"latitude": 37.94862722088389, "longitude": -91.78302701112852}, # Right Midpoint
+    # ]
+
+    drone: System = System()
+
+    # algorithm
+    # (2 coordinates for side 1) / 2 = starting point
+    # (2 coordinates for opposite side) / 2 = ending coordinate
+    # altitude = get_altitude_for_odlc(width, focal_length)
+    # altitude = 75 if altitude < 75 else altitude
+    # set drone height to altitude
+    # fly between starting and ending coordinates, this should fly it length wise
+    # run video camera the whole time recording data about odlc positions -- vision task?
+
+    # Add utm coordinates to all
+    data_search_area_boundary_utm: dict[str, float | int | str] = all_latlon_to_utm(
+        data_search_area_boundary
+    )
+
+    # Generate search path
+    search_paths: dict[str, float | int | str]
+    BUFFER_DISTANCE: int = -40  # use height/2 of camera image area on ground as buffer distance
+    search_paths = generate_search_paths(data_search_area_boundary_utm, BUFFER_DISTANCE)
```

Comment:
The generate_search_paths() function returns a list of tuple[float, float], so this type annotation is incorrect. Also, you can put the type annotation on the same line as  the assignment:

BUFFER_DISTANCE: int = -40
search_paths: list[tuple[float, float]] = generate_search_paths(data_search_area_boundary_utm, BUFFER_DISTANCE)